rust/src/ja4.rs

```diff
@@ -268,12 +299,46 @@ pub unsafe extern "C" fn SCJA4SetALPN(j: &mut JA4, proto: *const c_char, len: u1
     j.set_alpn(b);
 }
 
+#[no_mangle]
+pub unsafe extern "C" fn SCJA4GetVersion(j: &JA4) -> u16 {
+    u16::from(j.tls_version.unwrap_or(TlsVersion(0)))
+}
+
 #[no_mangle]
 pub unsafe extern "C" fn SCJA4GetHash(j: &mut JA4, out: &mut [u8; 36]) {
-    let hash = j.get_hash();
+    let hash = j.get_ja4_hash();
     out[0..36].copy_from_slice(hash.as_bytes());
 }
 
+#[no_mangle]
+pub unsafe extern "C" fn SCJA4SGetHash(j: &mut JA4, out: &mut [u8; 25]) {
```

Comment:
Can we have something like `#define JA4_LEN 25` ?

Reply:
Meaning it should be defined in rust and used from rust.h thanks to cbindgen ;-)